### ISSUE TITLE
feature request: persist queue

### ISSUE THREAD

**Issue by keegancsmith** [skeeto/youtube-dl-emacs#4]
Hi @skeeto and thanks so much for the great software. This request is to persist the queue to disk so that in the event of emacs crashing/power loss you don't lose your queue. For example I hadn't procesed my RSS feed in a while and built up a huge queue of videos to download. It was at this point my power tripped.

Is this relatively simple to add? I'm not an elisp expert, but I'd be happy to dive in and make an attempt at adding this if you are interested in it.

**Comment by stardiviner** [comment]
good idea. Might use elisp data file to store queue list. When finished downloading, auto delete entry in elisp data file.
